README.md

```diff
@@ -288,6 +288,7 @@ See the available options in the table below.
 | minRadiusCircleMarker | `null`                                | Set the min radius of a `CircleMarker` when editable is active.                                                                                                                                                 |
 | maxRadiusCircleMarker | `null`                                | Set the max radius of a `CircleMarker` when editable is active.                                                                                                                                                 |
 | editable              | `false`                               | Makes a `CircleMarker` editable like a `Circle`.                                                                                                                                                                |
+| radiusEdit            | `true`                                | Enables radius editing after drawing a `Circle`
```

Comment:
Why not using `editing` like for CircleMarker?

Reply:
PR started with using `editable` like CircleMarker does but I ran into an issue when trying to set the backwards compatibility value, with it then being set for both and leading to unwanted results. In the end, it seemed like it'd be best to separate them, in case anyone only wants one or the other.